react/new/src/contexts/ThemeContext.tsx

```diff
@@ -0,0 +1,156 @@
+import {
+  createContext,
+  useContext,
+  useCallback,
+  useState,
+  ReactNode,
+} from 'react'
+import {
+  MantineProvider,
+  createTheme,
+  type MantineColorsTuple,
```

Comment:
why "type" here -- I've never seen that syntax before (looks ts-specific to me?)

Reply:
Yeah this is TS specific. It's to specify that you're importing a type, not a function or value, from the lib.

Reply:
Is it just a readability thing? Like if you left it out this would still run, but it's easier to understand?